rosdiagnostic/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rosdiagnostic)
+
+find_package(catkin REQUIRED COMPONENTS)
+
+catkin_package()
+
+catkin_python_setup()
+
+install(PROGRAMS
+  scripts/rosdiagnostic
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
```

Comment:
Newline please, and you can drop `COMPONENTS` from `find_package` above, since you haven't got any.